CSSharpUtils/Extensions/PlayerControllerExtensions.cs

```diff
@@ -182,6 +183,20 @@ public static void SetMoney(this CCSPlayerController? playerController, int mone
         Utilities.SetStateChanged(playerController, "CCSPlayerController", "m_pInGameMoneyServices");
     }
 
+    /// <summary>
+    /// Checks if the specified controller has a permission
+    /// </summary>
+    /// <param name="playerController">The player controller to check.</param>
+    /// <param name="permission">The permission to check.</param>
+    /// <returns><c>true</c> if the player has the permission; otherwise, <c>false</c>.</returns>
+    public static bool HasPermission(this CCSPlayerController? playerController, string permission)
+    {
+        if (!playerController.IsPlayer())
```

Comment:
this could (and probably should) be combined into one line.
`return playerController.IsPlayer() && AdminManager.PlayerHasPermissions(playerController, permission)`